homeassistant/components/media_player/__init__.py

```diff
@@ -97,7 +97,6 @@
     SERVICE_MEDIA_STOP: 'media_stop',
     SERVICE_MEDIA_NEXT_TRACK: 'media_next_track',
     SERVICE_MEDIA_PREVIOUS_TRACK: 'media_previous_track',
-    SERVICE_SELECT_SOURCE: 'select_source',
```

Comment:
It has its own service callback, see line 358. The dict is only for services that don't take an attribute as argument for the method. We could simplify all this and include all services in the dict and make one callback for all services, but I didn't want to do that right now.